packages/next/test/fixtures/00-app-dir-ppr-full/index.test.js

```diff
@@ -19,6 +19,9 @@ const pages = [
   { pathname: '/no-suspense/nested/a', dynamic: true },
   { pathname: '/no-suspense/nested/b', dynamic: true },
   { pathname: '/no-suspense/nested/c', dynamic: true },
+  { pathname: '/no-fallback/a', dynamic: true },
+  { pathname: '/no-fallback/b', dynamic: true },
+  { pathname: '/no-fallback/c', dynamic: true },
```

Comment:
Would be good to have a test case to ensure `/no-fallback/non-existent` `404s` properly and doesn't invoke the function.